pkg/kwokctl/cmd/delete/delete.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package delete
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kwok/pkg/kwokctl/cmd/delete/cluster"
+	"sigs.k8s.io/kwok/pkg/logger"
+)
+
+// NewCommand returns a new cobra.Command for cluster creation
+func NewCommand(logger logger.Logger) *cobra.Command {
```

Comment:
FYI: logs when deleting a running cluster using binary as the runtime:

```
⇒  kwokctl delete cluster
Stopping cluster "kwok-kwok"
failed to kill kwok: wait: no child processes
failed to kill kube-scheduler: wait: no child processes
failed to kill kube-controller-manager: wait: no child processes
failed to kill kube-apiserver: wait: no child processes
failed to kill etcd: wait: no child processes
Deleting cluster "kwok-kwok"
Cluster "kwok-kwok" deleted
```

Are the "failed to kill" strings as expected?